ccd-definition/CaseEvent/User/UserEvents.json

```diff
@@ -82,7 +82,7 @@
     "PreConditionState(s)": "PENDING_CASE_ISSUED;CASE_ISSUED;AWAITING_CASE_DETAILS_NOTIFICATION;AWAITING_RESPONDENT_ACKNOWLEDGEMENT;AWAITING_APPLICANT_INTENTION",
     "PostConditionState": "*",
     "SecurityClassification": "Public",
-    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-start",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/version/V_1/about-to-start",
```

Comment:
This would need to go behind a nonprod toggle, as we dont want to enable your callback just yet in production. Until MP goes live, it should remain as is (Adding CreatedDate + Date for LitigantFriend1)